.travis.yml

```diff
@@ -6,7 +6,7 @@ cache:
 notifications:
   email: false
 node_js:
-  - '4'
+  - stable
   - '0.10'
 before_install:
   - npm i -g npm@^3.0.0
```

Comment:
I'd definitely test on `npm@^2.0.0` too. There seems to be a whole lot of devs still using not only `node@0.1x`, but also `npm@^2.0.0`. On `ghooks` I introduced [a script that decides which version of npm to install based on the current node version](https://github.com/ta2edchimp/ghooks/blob/master/.travis.yml#L16-L20). (This is mainly because of something I can only describe as a bug, in `npm@2.15.x` and I think it would be good to cover such cases as early as possible.)